src/components/contact/appsco-manage-contact-components-page.html

```diff
@@ -95,6 +95,7 @@
             .groups {
                 @apply(--layout-vertical);
                 margin-top: 10px;
+                margin-bottom: -10px;
```

Comment:
why negative margin? If it can be achieved without using negative margin that would be awesome.
Please consult @NikolaVu on this subject